libexec/alluxio-config.sh

```diff
@@ -151,6 +151,12 @@ if [[ -n "${ALLUXIO_LOGSERVER_HOSTNAME}" && -n "${ALLUXIO_LOGSERVER_PORT}" ]]; t
     ALLUXIO_WORKER_JAVA_OPTS+=" -Dalluxio.remote.logger.type=REMOTE_WORKER_LOGGER"
 fi
 
+# FUSE specific parameters that will be shared to all workers based on ALLUXIO_JAVA_OPTS.
+ALLUXIO_FUSE_JAVA_OPTS_DEFAULT=" -server -Xms1G -Xmx1G -XX:MaxDirectMemorySize=4g"
+ALLUXIO_FUSE_JAVA_OPTS_DEFAULT+=" -Djava.library.path=${ALLUXIO_HOME}/lib"
```

Comment:
that's sounds very nice to have. here I see it is not very necessary as it will complicate substaintially the script here.
I would leave it when we see people are encounting or asking for this

Reply:
Sounds good!